discord/ext/menus/pagination.py

```diff
@@ -0,0 +1,160 @@
+import discord, asyncio
+from typing import List, Union
+
+from discord import abc
+from discord.interactions import Interaction
+from discord.utils import MISSING
+from discord.ui.button import button
+from discord.commands import ApplicationContext
+from discord.ext.commands import Context
+
+
+class Paginate(discord.ui.View):
+    """Creates a paginator for embeds that is navigated with buttons.
```

Comment:
```suggestion
    """Creates a paginator for a message that is navigated with buttons.
```
Text content works too